internal/datastore/crdb/schema/indexes_test.go

```diff
@@ -26,16 +26,16 @@ func TestIndexForFilter(t *testing.T) {
 			name:                     "filter by resource type",
 			filter:                   datastore.RelationshipsFilter{OptionalResourceType: "foo"},
 			expectedWithoutIntegrity: "pk_relation_tuple",
-			expectedWithIntegrity:    "pk_relation_tuple",
+			expectedWithIntegrity:    "ix_relation_tuple_with_integrity",
 		},
 		{
 			name: "filter by resource type and relation",
 			filter: datastore.RelationshipsFilter{
 				OptionalResourceType:     "foo",
 				OptionalResourceRelation: "bar",
 			},
-			expectedWithoutIntegrity: "pk_relation_tuple",
-			expectedWithIntegrity:    "pk_relation_tuple",
```

Comment:
Are these changes reflecting a change in behavior of index selection because the structure of the query doesn't actually match the structure of the index under this new work?